packages/block-editor/src/components/rich-text/index.native.js

```diff
@@ -237,10 +237,12 @@ export class RichText extends Component {
 	 * Handles any case where the content of the AztecRN instance has changed
 	 */
 	onChange( event ) {
-		this.lastEventCount = event.nativeEvent.eventCount;
-		const contentWithoutRootTag = this.removeRootTagsProduceByAztec( unescapeSpaces( event.nativeEvent.text ) );
-		this.lastContent = contentWithoutRootTag;
-		this.props.onChange( this.lastContent );
+		if ( event.nativeEvent ) {
+			this.lastEventCount = event.nativeEvent.eventCount;
+			const contentWithoutRootTag = this.removeRootTagsProduceByAztec( unescapeSpaces( event.nativeEvent.text ) );
+			this.lastContent = contentWithoutRootTag;
+			this.onChange( this.lastContent );
```

Comment:
It should be `this.props.onChange` here.

Reply:
👍 addressed in f87fae612